zanata-war/src/test/resources/arquillian.xml

```diff
@@ -34,10 +31,7 @@
   <container qualifier="wildfly81">
     <configuration>
       <!--<property name="jbossHome">${wildfly.home}</property>-->
-      <!--<property-->
-        <!--name="javaVmArguments">-Xms1536m -Xmx1536m -XX:MaxPermSize=256m -Dorg.jboss.as.logging.per-deployment=false-->
-      <!--</property>-->
-
+      <property name="javaVmArguments">-Dorg.jboss.as.logging.per-deployment=false</property>
```

Comment:
It think that system property has actually been deprecated in WildFly - https://issues.jboss.org/browse/WFLY-2600, but I'm not quite sure what it's meant to be replaced by. See https://docs.jboss.org/author/display/WFLY10/Logging+Configuration#LoggingConfiguration-Attributes

For now, let's just keep it as a comment, as before.